packages/sanity/src/core/form/studio/tree-editing/components/breadcrumbs/TreeEditingBreadcrumbsMenuButton.tsx

```diff
@@ -71,7 +78,7 @@ export function TreeEditingBreadcrumbsMenuButton(
     selectedPath,
   } = props
   const [open, setOpen] = useState<boolean>(false)
-  const [rootElement, setRootElement] = useState<HTMLElement | null>(null)
+  const rootElementRef = useRef<HTMLDivElement | null>(null)
   const [buttonElement, setButtonElement] = useState<HTMLButtonElement | null>(null)
```

Comment:
Could this also be turned into a ref now?

Reply:
[Probably](https://github.com/sanity-io/sanity/blob/bedbd5ad561aa7eab0324038dd2f49f708a86d19/packages/sanity/src/core/form/studio/tree-editing/components/breadcrumbs/TreeEditingBreadcrumbsMenuButton.tsx#L88)